mmdeploy/codebase/mmdet/deploy/object_detection.py

```diff
@@ -266,6 +266,10 @@ def get_preprocess(self, *args, **kwargs) -> Dict:
             and 'Annotation' not in item['type']
         ]
         for i, transform in enumerate(transforms):
+            # deal with mmyolo
+            if transform['type'].startswith('mmdet.'):
+                transform['type'] = transform['type'][6:]
+                transforms[i]['type'] = transform['type']
```

Comment:
this is redundant?

Reply:
`transform['type']` is used later in the for loop and it should be updated as well.